### PR TITLE
Internal: Fixed browser-import JS Lint warnings [ED-6572]

### PR DESCRIPTION
**Pull request by YitzhakAvm**

### CODE REVIEW

File: assets/dev/js/editor/components/browser-import/files/file-parser-base.js [hunk 1]
```diff
@@ -60,9 +58,12 @@ export default class FileParserBase {
 	 * very accurate and specific so if the parser can't handle the file for sure, the next parsers will have the
 	 * opportunity to do so.
 	 *
+	 * @param reader
+	 *
 	 * @abstract
 	 * @returns {boolean}
 	 */
+	// eslint-disable-next-line no-unused-vars
 	static async validate( reader ) {
```
Comment (iNewLegend) [inline assets/dev/js/editor/components/browser-import/files/file-parser-base.js:67]:
```suggestion
	static async validate( reader ) {  // eslint-disable-line no-unused-vars
```

File: core/app/assets/js/molecules/tooltip.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export default function Tooltip( props ) {
 				isAborted.current = true;
 
 				// Cleanup of existing tipsy element in case of re-render.
+				// eslint-disable-next-line no-jquery/no-sizzle
```
Comment (matipojo) [inline core/app/assets/js/molecules/tooltip.js:63]: Why? just don't jQuery

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/components/browser-import/files/file-parser-base.js (hunk 1 above)
Comment (ItzhakAvm) [reply assets/dev/js/editor/components/browser-import/files/file-parser-base.js]: Fixed.

File: core/app/assets/js/molecules/tooltip.js (hunk 2 above)
Comment (ItzhakAvm) [reply core/app/assets/js/molecules/tooltip.js]: Fixed.

**Review by iNewLegend — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please start from and send to 3.6.0

**Comment by ItzhakAvm** [comment]
> Please start from and send to 3.6.0

@matipojo Done.

**Pull request merged**